src/commands/Register.ts

```diff
@@ -136,8 +149,8 @@ export async function executeRegister(registerOptions?: RegisterOptions) {
       : {
           hub: contracts.relayHubAddress,
           signer: await register.findWealthyAccount(),
```

Comment:
```suggestion
          signer: account  ?? await register.findWealthyAccount(),
```

Reply:
ah yea I forgot to add this. However, this wont work as signer is an object, so it will need to be constructed. That should be simple given we have the provider, however this exposes some issues in the CommandClient. Perhaps best we have a chat about it.

Reply:
we had a discussion and another task for this needs to be created

Reply:
after discussion, resolving by removing the class altogether and building the account within this process

Reply:
done